react/components/BankInvoice/Embedded.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC, useState } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { ButtonWithIcon, IconCaretDown, IconCaretUp } from 'vtex.styleguide'
+
+interface Props {
+  url: string
+}
+
+const Embedded: FC<Props> = ({ url }) => {
+  const [isOpen, setIsOpen] = useState(false)
```

Comment:
```suggestion
  const [isOpen, setIsOpen] = useState<boolean>(false)
  
  const toggle = () => setIsOpen(!isOpen)
```

Reply:
No need for that, it correctly infers the state type using the `initialState` value:

![image](https://user-images.githubusercontent.com/12702016/71524721-9146a200-28ad-11ea-950f-5eca0404db31.png)

Reply:
@kaisermann all right, but use `toggle` function.